pkgs/servers/plexpy/default.nix

```diff
@@ -0,0 +1,41 @@
+{stdenv, fetchFromGitHub, python}:
+
+stdenv.mkDerivation rec {
+  version = "1.4.25";
+  pname = "plexpy";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "JonnyWong16";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0a4ynrfamlwkgqil4n61v47p21czxpjdzg0mias4kdjam2nnwnjx";
+  };
+
+  phases = "unpackPhase installPhase fixupPhase";
```

Comment:
Don't set phases. Instead, if you do not need a certain phase, you could write `buildPhase = ":";`.